docs/docs/getting-started/bare-metal.md

```diff
@@ -0,0 +1,18 @@
+# Prepare a Bare Metal Instance
+
+## Hardware & Firmware Setup
+
+1. Update your BIOS to a version that supports AMD SEV-SNP. Updating to the latest available version is recommended as newer versions will likely contain security patches for AMD SEV-SNP.
+2. Enter BIOS setup to enable SMEE, IOMMU, RMP coverage, and SEV-SNP. Set the SEV-ES ASID Space Limit to a non-zero number (higher is better).
+3. Download the latest firmware version for your processor from [AMD](https://www.amd.com/de/developer/sev.html), unpack it, and place it in `/lib/firmware/amd`.
+
+Consult AMD's [Using SEV with AMD EPYC Processors](https://www.amd.com/content/dam/amd/en/documents/epyc-technical-docs/tuning-guides/58207-using-sev-with-amd-epyc-processors.pdf) user guide for more information.
+
+## Kernel Setup
+
+1. Install a kernel with version 6.11 or greater.
```

Comment:
That's not released yet, right? Can we give an rc version that works?

Reply:
> That's not released yet, right?

No, not yet, although 6.11 will likely be released in less than two weeks from now.

> Can we give an rc version that works?

No, we can't because of the regression in 6.11-rc4 that won't be fixed until 6.11-rc7 (not yet released). We could mention 6-11-rc3 although that might be confusing because users might expect that the newer rcs will work as well.

Reply:
We can add exactly that to the documentation.